apimonitor/models.py

```diff
@@ -118,3 +118,18 @@ class AlertsConfiguration(models.Model):
     email_password = models.CharField(max_length=1024, blank=True, default="")
     email_use_tls = models.BooleanField(default=False)
     email_use_ssl = models.BooleanField(default=False)
+
+    # Threshold config
+    time_window_choices = [
+        ('1H', '1 Hour'),
+        ('2H', '2 Hour'),
+        ('3H', '3 Hour'),
+        ('6H', '6 Hour'),
+        ('12H', '12 Hour'),
+        ('24H', '24 Hour')
+    ]
+    threshold_pct = models.IntegerField(null=True, blank=True, default=100, validators=[
+        MinValueValidator(1),
+        MaxValueValidator(100)
+    ])
```

Comment:
```suggestion
    threshold_pct = models.IntegerField(default=100, validators=[
        MinValueValidator(1),
        MaxValueValidator(100)
    ])
```

ini kayaknya jangan null=True, blank=True kali ya